cobalt/dom/source_buffer_metrics.cc

```diff
@@ -121,21 +121,33 @@ void SourceBufferMetrics::EndTracking(SourceBufferMetricsAction action,
 #endif  // !defined(COBALT_BUILD_TYPE_GOLD)
 }
 
-void SourceBufferMetrics::RecordTelemetry(
-    SourceBufferMetricsAction action, const base::TimeDelta& action_duration) {
+void SourceBufferMetrics::RecordUMATelemetry(
+    SourceBufferMetricsAction action, const base::TimeDelta& action_duration,
+    size_t size_appended) {
   switch (action) {
     case SourceBufferMetricsAction::PREPARE_APPEND:
       UMA_HISTOGRAM_CUSTOM_MICROSECONDS_TIMES(
           "Cobalt.Media.SourceBuffer.PrepareAppend.Timing", action_duration,
           base::TimeDelta::FromMicroseconds(1),
           base::TimeDelta::FromMilliseconds(25), 50);
       break;
-    case SourceBufferMetricsAction::APPEND_BUFFER:
+    case SourceBufferMetricsAction::APPEND_BUFFER: {
       UMA_HISTOGRAM_CUSTOM_MICROSECONDS_TIMES(
           "Cobalt.Media.SourceBuffer.AppendBuffer.Timing", action_duration,
           base::TimeDelta::FromMicroseconds(1),
           base::TimeDelta::FromMilliseconds(25), 50);
+
+      UMA_HISTOGRAM_COUNTS_10M(
+          "Cobalt.Media.SourceBuffer.AppendBuffer.BytesAppended",
+          size_appended);
+
+      int64_t bytes_per_microsecond =
+          size_appended / action_duration.InMicroseconds();
```

Comment:
Consider guarding `action_duration` against 0.  Also, consider moving the extra pair of {} around the calculation and log of `bytes_per_microsecond` so it's more "local".

Reply:
Done.